cf_remote/remote.py

```diff
@@ -208,8 +209,17 @@ def get_info(host, *, users=None, connection=None):
     else:
         data["os"] = "unix"
 
-        scp("nt-discovery.sh", host, connection, hide=True)
-        discovery = parse_envfile(ssh_sudo(connection, "bash nt-discovery.sh"))
+        cf_remote_dir = dirname(__file__)
+        script_path = join(cf_remote_dir, "nt-discovery.sh")
+        stat(script_path)
```

Comment:
What is this `stat()` good for? Is it a poor-man's `os.path.exists()`?

Reply:
Yes, it was to error out if the file did not exist. I can re-work.